packages/web-components/src/storybook-typings.d.ts

```diff
@@ -2,3 +2,8 @@ declare module '*.html' {
   const value: string;
   export default value;
 }
+
+declare module '*.svg' {
```

Comment:
It is not clear that the `packages/web-components/src/divider/paginated_reports_20_regular.svg` is used just in the story - that would confuse contributors but also the svg would remain in the NPM package.

Either define the SVG inline in the stories or rename it to `.stories.svg`.

Reply:
I'd agree with inline for the time being